src/main/scala/org/alvearie/hri/flink/core/MgmtApiSink.scala

```diff
@@ -31,7 +31,14 @@ class MgmtApiSink(val tenantId: String, val mgmtApiUrl: String, val mgmtClientId
   @transient lazy val mgmtClient: MgmtClient = createMgmtClient()
 
   // This enables overriding for testing
-  def createMgmtClient(): MgmtClient = new MgmtClient(mgmtApiUrl, mgmtClientId, mgmtClientSecret, mgmtClientAudience, oauthServiceBaseUrl)
+  def createMgmtClient(): MgmtClient = {
+    try {
+      new MgmtClient(mgmtApiUrl, mgmtClientId, mgmtClientSecret, mgmtClientAudience, oauthServiceBaseUrl)
+    } catch {
+      case ex: Throwable =>
+        throw new FlinkException(ex) // can't recover from this
+    }
+  }
```

Comment:
Why this change? I think this was part of changes to v3.0 that don't need to be in v2.x

Reply:
Slipped by me, it's reverted now ✅